crates/winmd/src/types/delegate.rs

```diff
@@ -189,10 +189,8 @@ impl Delegate {
                     }
                 }
                 #invoke_sig {
-                    unsafe {
-                        let this: *mut Self = this.as_raw() as _;
-                        ((*this).invoke)(#(#invoke_args,)*).into()
-                    }
+                    let this: *mut Self = this.as_raw() as _;
+                    ((*this).invoke)(#(#invoke_args,)*).into()
```

Comment:
The beauty of the `build` macro and then including the source, beyond debuggability and code completion, is that we can pick up compiler warnings that are otherwise silent when only a `TokenStream` is used (as with the `import` macro). In this case, the `unsafe` was redundant.